brainvisa/toolboxes/tools/processes/assemblynet.py

```diff
@@ -120,5 +127,6 @@ def execution(self, context):
                 file_path.rename(output_file)
 
     # Update brainvisa database to take into account results
-    db = databases.database(self.output_folder.get('_database'))
-    db.update(directoriesToScan=[self.output_folder.fullPath()])
+    # Commented for now, otherwise we can't run it in parallel mode. TODO Find an other way
+    # db = databases.database(self.output_folder.get('_database'))
+    # db.update(directoriesToScan=[self.output_folder.fullPath()])
```

Comment:
Maybe we can put those line inside a try/except, in order to let those lines work in brainvisa, but not raise an error in case of running on cluster?

Reply:
Good idea, I test it and it works, thanks ! :)